bigquery/indexer.py

```diff
@@ -168,29 +175,18 @@ def index_table(es, index_name, table_name, participant_id_col, sample_id_col,
             (only needed on samples tables).
         sample_file_cols: (optional) Mappings for columns which contain genomic
             files of a particular type (specified in ui.json).
-        billing_project_id: GCP project ID to bill for reading table
     """
-    _create_nested_mappings(es, index_name, table_name, sample_id_col,
-                            billing_project_id)
-
+    _create_nested_mappings(es, index_name, table, sample_id_col)
+    table_name = table.full_table_id
     start_time = time.time()
     logger.info('Indexing %s.' % table_name)
 
-    # There is no easy way to import BigQuery -> Elasticsearch. Instead:
-    # BigQuery table -> pandas dataframe -> dict -> Elasticsearch
-    df = pd.read_gbq(
-        'SELECT * FROM `%s`' % table_name,
-        project_id=billing_project_id,
-        dialect='standard')
-    elapsed_time = time.time() - start_time
-    elapsed_time_str = time.strftime('%Hh:%Mm:%Ss', time.gmtime(elapsed_time))
-    logger.info('BigQuery -> pandas took %s' % elapsed_time_str)
-    logger.info('%s has %d rows' % (table_name, len(df)))
+    df = bq.list_rows(table).to_dataframe()
```

Comment:
Put back this comment, it explains why we're making a df.
```
# There is no easy way to import BigQuery -> Elasticsearch. Instead:
# BigQuery table -> pandas dataframe -> dict -> Elasticsearch
```

Reply:
Done